### PR TITLE
web-api-fetch

### PR DESCRIPTION
**Pull request by ananthmgk**
my next 5 web API fetching html

### CODE REVIEW

**Comment by ananthmgk** [comment]
Added bird sound and emoji symbols

**Comment by ananthmgk** [comment]
bird sound and emoji

**Comment by ananthmgk** [comment]
created isro with increment item no. and created thirukkural with increment url.

File: cat-facts.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      
+      </script>
+  </head>
+  <body>
+    <div>
+      <h1 id="heading">Cat Facts</h1>
+      <span id="text"></span>
+    </div>
+    <div>
+        <button onClick="window.location.reload()">Next</button>
```
Comment (ramo) [inline cat-facts.html]: Instead of windows reload, can we achieve this simply using AJAX?

**Review by ramo — CHANGES_REQUESTED** [review]
Please change the reloading of the windows to refresh the content. Please install prettier plugin and ensure the same formatting throughout the project. 
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

**Comment by ananthmgk** [comment]
corrected all the files

**Review by ramo — APPROVED** [review]

**Pull request merged**